docs/users-guide/interactive-dashboards.md

```diff
@@ -0,0 +1,104 @@
+
+![Click behavior menu](images/interactive-dashboards/click-behavior-menu.png)
+
+### Open the action menu
+
+For questions composed using the query builder, the default click behavior is to open the **action menu**, which presents people with the option to [drill through the data](https://www.metabase.com/blog/drilling-through-data/index.html):
+
+![Action menu](images/interactive-dashboards/action-menu.png)
+
+### Custom destinations
+
+You can set custom destinations for all questions, including SQL questions.
+
+![Custom destinations](images/interactive-dashboards/custom-destination.png)
+
+Possible destinations include:
+
+- Dashboards
+- Saved questions
+- URLs
+
+### Passing values to the destination
+
+If you're linking to a dashboard or a SQL question that has filters, you can pass values from the current dashboard to filters in the destination. 
+
+For example, if you link to a dashboard that has a filter for `Category`, you can pass a value for `Category` from the origin question to the destination dashboard:
+
+![Pass value to dashboard](images/interactive-dashboards/pass-value.png)
+
+Once you select a column, the sidebar will display the column used to pass the value, as well as the target filter at the destination that Metabase will pass the value to.
+
+![Pass category to filter](images/interactive-dashboards/pass-category-to-filter.png)
+
+In the example above, when a user clicks on the **Orders by product category** card, Metabase will pass the clicked `Category` to the destination dashboard ("Interactive Dashboard"), which will then filter its cards by that `Category`.
+
+You can also send the currently selected value of a dashboard filter on the current dashboard to the destination. And if you're using [Metabase Enterprise Edition](https://www.metabase.com/enterprise/scale/index.html), you can pass a user attribute provided by SSO to the destination, too. Those user attributes will show up as options when you click on one of the destination's filters ( provided the values are compatible with that filter).
+
+When displaying questions as tables, you can select different click behaviors for different columns in the table. You can also modify the contents of the cells in a given column, replacing the value with custom text. For example, if you had a column that listed categories, you could change the text in the cell to read: "Click for details about {{Category}}".
+
+You can also use values to construct URLs to external resources.
+
+![Enter a URL](images/interactive-dashboards/enter-a-url.png)
+
+From the **Click behavior** sidebar, select **Go to a custom destination** and link to **URL**. The **Enter a URL to link to** modal will pop up, allowing you to specify a URL, as well as specify the value of a column or dashboard filter. Click on the dropdown menu **Values you can reference** to see your options.
```

Comment:
Feels like we could give an example URL here since the syntax to pass values is a little tricky, especially when passing multiple values. The old EE doc, `enterprise-guide/customizing-drill-through.html`, has some examples that we could lift.

Reply:
That's a good call. I was relying on the screenshot, but I should reinforce in the text.